# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lua-codegen/destdir.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2011-06-08 16:36:56.000000000 +0200
+++ ./Makefile	2011-06-08 16:38:28.000000000 +0200
@@ -6,19 +6,17 @@
   REV   := 1
 endif
 
-ifndef DESTDIR
-  DESTDIR := /usr/local
-endif
-LIBDIR  := $(DESTDIR)/share/lua/5.1
+prefix  = /usr/local
+LIBDIR  = $(prefix)/share/lua/5.1
 
 install:
-	mkdir -p $(LIBDIR)/CodeGen
-	cp src/CodeGen.lua              $(LIBDIR)
-	cp src/CodeGen/Graph.lua        $(LIBDIR)/CodeGen
+	mkdir -p $(DESTDIR)$(LIBDIR)/CodeGen
+	cp src/CodeGen.lua              $(DESTDIR)$(LIBDIR)
+	cp src/CodeGen/Graph.lua        $(DESTDIR)$(LIBDIR)/CodeGen
 
 uninstall:
-	rm -f $(LIBDIR)/CodeGen.lua
-	rm -f $(LIBDIR)/CodeGen/Graph.lua
+	rm -f $(DESTDIR)$(LIBDIR)/CodeGen.lua
+	rm -f $(DESTDIR)$(LIBDIR)/CodeGen/Graph.lua
 
 manifest_pl := \
 use strict; \
